### ISSUE TITLE
Callable functions internal error

### ISSUE THREAD

**Issue by kubanac95** [firebase/firebase-js-sdk#3849]
### [REQUIRED] Describe your environment

  * Operating System version: Windows 10 Pro - 1909
  * Browser version: Chrome - 85.0.4183.121
  * Firebase SDK version: 7.21.1
  * Firebase Product: functions

### [REQUIRED] Describe the problem

I have been working on a chrome extension trying to integrate firebase js SDK for authentication alongside other services.

I made the authentication working fine as well as firestore for reading out documents, but when I tried calling a fireabse callable function I tun into some strange issues.

SDK kept returning an error with `{ "code": "internal" }`

![image](https://user-images.githubusercontent.com/16191931/94349689-319d4300-0047-11eb-9d9f-ce6728b91e4f.png)

At first, I taught it was an issue with my implementation, that I'm missing some permissions or CSP since it was working in my web app. After hours of scratching my head, I decided to try a different firebase SDK version.

Strangely enough, that fixed the problem. Going down from `7.21.1` to `7.21.0` resolved the issues and the internal error lo longer occurs.

I think this will be enough to pinpoint the issue since it seems to be a transition between those two versions that caused the issue.

Code to 

#### Relevant Code:
    firebase
      .functions()
      .httpsCallable("onCreateSearch")({
        url: "https://github.com/firebase/firebase-js-sdk",
      })
      .then(console.log)
      .catch((error) => {
        console.log(error);
        console.log(JSON.stringify(error));
      });

**Comment by hsubox76** [comment]
Seems likely to be related to PR https://github.com/firebase/firebase-js-sdk/pull/3782

Will take a look.

**Comment by hsubox76** [comment]
For my own notes - because I did not add a changeset to #3782, @firebase/functions was not released with 7.21.1, but the yarn.lock file with isomorphic-fetch removed would have been. So the dep was removed but the code using it was not updated. 7.22.0 did publish the functions package because it included a changeset for the customDomain change, and consequently introduced the bind error.

TLDR, both 7.21.1 and 7.22.0 will have functions callable errors, for different reasons.
